### PR TITLE
trivial: fix linter errors and return verbose errors

### PR DESCRIPTION
**Pull request by yyforyongyu**
Fixed complaints from IDE and linters, and improved returned errors when creating this [analysis](https://github.com/lightningnetwork/lnd/issues/7261#issuecomment-2153008660).

### CODE REVIEW

File: peer/brontide.go [hunk 1]
```diff
@@ -2938,7 +2938,7 @@ func (p *Brontide) restartCoopClose(lnChan *lnwallet.LightningChannel) (
 		})
 
 	// An error other than ErrNoShutdownInfo was returned
-	case err != nil && !errors.Is(err, channeldb.ErrNoShutdownInfo):
+	case !errors.Is(err, channeldb.ErrNoShutdownInfo):
```
Comment (bhandras) [inline peer/brontide.go:2941]: I think the nilness check is still needed here as `!errors.Is(err, ...)` is true if `err == nil`.

Comment (yyforyongyu) [reply peer/brontide.go:2941]: I was a bit surprised too but then realized the `err == nil` case is caught above in the `case err == nil:`.

Comment (bhandras) [reply peer/brontide.go:2941]: Indeed!

**Review by bhandras — APPROVED** [review]
~LGTM :tada:~ Redacting, I realized that tests are still failing.

File: channeldb/graph.go [hunk 2]
```diff
@@ -1349,7 +1349,7 @@ func (c *ChannelGraph) PruneGraph(spentOutputs []*wire.OutPoint,
 				edges, edgeIndex, chanIndex, zombieIndex,
 				chanID, false, false,
 			)
-			if err != nil && err != ErrEdgeNotFound {
+			if err != nil && !errors.Is(err, ErrEdgeNotFound) {
```
Comment (bhandras) [inline channeldb/graph.go:1352]: I think we generally need to be careful with `error.Is` as it is much much slower than just a simple equivalency check.

**Review by bhandras — APPROVED** [review]
LGTM 🌮

**Pull request merged**